test/PixelRendr/encode.ts

```diff
@@ -15,10 +15,10 @@ mochaLoader.addTest("processes the image into a string", (): void => {
     var sprite = PixelRender.encode(img);
 
     // Assert
-    chai.expect(sprite).to.equal("x16");
+    chai.expect(sprite).to.equal("x16,");
 });
 
-mochaLoader.addTest("calls the callback", (): void => {
+mochaLoader.addTest("calls the callback passed to encode once", (): void => {
```

Comment:
"encode calls the callback passed to encode once"

You could remove "passed to encode" and reword slightly to be more consise.